core/_cli_utils.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import click
+from . import utils
+
+
+logger = logging.getLogger(__name__)
+
+class opts:
+    layout = click.option(
+        "--layout", default=None, type=click.Choice(list(utils.ANTS_DICT.keys())),
+        help="A pre-defined HERA layout to use"
+    )
+    ants = click.option(
+        "-a", "--ants", type=click.IntRange(0, 350), default=None, multiple=True,
```

Comment:
Is this equal to a closed interval of [0, 349] or [0, 350]. The latter would total to 351 antennas right?

Reply:
Good catch, I fixed this now